### PR TITLE
Add recover password flow in app

### PR DESCRIPTION
**Pull request by VitaliKozhurov**

### CODE REVIEW

**Review by DenisKulik — APPROVED** [review]
👍👍👍

File: src/features/auth/ui/checkEmail/CheckEmail.tsx [hunk 1]
```diff
@@ -20,7 +22,7 @@ export const CheckEmail = ({ email = 'example@mail.com' }: CheckEmailProps): JSX
       <Typography className={s.instructionText} variant={TypographyVariant.Body2}>
         {`We’ve sent an Email with instructions to ${email}`}
       </Typography>
-      <Button className={s.signInLink} as="a" href="https://google.com" fullWidth>
+      <Button className={s.signInLink} as={Link} to="/sign-in" fullWidth>
```
Comment (vitalyabedik) [inline src/features/auth/ui/checkEmail/CheckEmail.tsx:25]: use enum for route

Comment (VitaliKozhurov) [reply src/features/auth/ui/checkEmail/CheckEmail.tsx]: Done!

**Review by vitalyabedik — APPROVED** [review]
wow!

**Pull request merged**